mmgen/models/gans/base_translation_model.py

```diff
@@ -0,0 +1,200 @@
+from copy import deepcopy
+
+import torch.nn as nn
+
+from ..builder import MODELS, build_module
+from .base_gan import BaseGAN
+
+
+@MODELS.register_module()
+class BaseTranslationModel(BaseGAN):
+    """Basic translation Model.
+
+    Args:
+        generator (dict): Config for the generator.
+        discriminator (dict): Config for the discriminator.
+        gan_loss (dict): Config for the gan loss.
+        default_style (str): Default output style.
+        reachable_styles (list[str], optional): Styles that can be generated by
+            the model. Defaults to [].
```

Comment:
Typically, we will not use `[]` or `{}` as the default argument. Can we use `None` as the default value?